tests/basic_multivector_grpc_test.sh

```diff
@@ -242,4 +242,42 @@ if [[ $response != *"vectors count must be greater than 0"* ]]; then
     exit 1
 fi
 
-set -e
\ No newline at end of file
+# search fails if the dense vector is not of the right dimension
+response=$(
+  $docker_grpcurl -d '{
+      "collection_name": "test_multivector_collection",
+      "vector": [0.2,0.1,0.9],
+      "limit": 3,
+      "vector_name": "my-multivec"
+    }' $QDRANT_HOST qdrant.Points/Search 2>&1
+)
+
+if [[ $response != *"Wrong input: Vector dimension error: expected dim: 4, got 3"* ]]; then
+    echo Unexpected response, expected validation error: "$response"
+    exit 1
+fi
+
+response=$(
```

Comment:
Do we actually need to check `dim=3` and `dim=5` both? I find it equal

Reply:
No, just wanted to make sure less and greater values were handled.

I can remove it if it bugs you :)

Reply:
Honestly, it bugs me a little bit. But it's so small nitpick that you may leave it as is

Reply:
https://github.com/qdrant/qdrant/pull/4203/commits/f714a8ef2def4bd0b0636d29b86c3e5a155e0734